sdks/java/client-java-core/CHANGELOG.adoc

```diff
@@ -0,0 +1,7 @@
+=== Changes
```

Comment:
could we match to the typescript changelog pls

Reply:
done